### DISCUSSION TITLE
Want to fetch fresh page data on every navigation using link

### DISCUSSION THREAD

**Discussion by danieljee** [vercel/next.js#52760] [Help] [answered]
### Summary

Let's say I have an item page (/items/[itemId]/page.tsx) that fetches data from an API and displays the information to the user:
```
export default async function ItemPage({ params }: { params: { itemId: string }}){

  const item = await fetchItem(itemId);

  return (
    <div>
      <div>{item.name}</div>
      <div>{item.description}</div>
    </div>
  )

}
```
Whenever a user navigates to this page using <link> on the client side, the data should always be up to date.
However, what happens is that the data is stale until I do a complete refresh.

Is it possible with app router to achieve the same behaviour as using getServerSideProps in page router?

Thanks

**Comment by masterkain** [comment, marked as answer]
there are some tickets open about this, some with tag linear, so hopefully we should see it fixed at some point

https://github.com/vercel/next.js/issues/52727
https://github.com/vercel/next.js/issues/42991
https://github.com/vercel/next.js/issues/51788
